po/it_IT.po

```diff
@@ -0,0 +1,1554 @@
+msgstr "Annullo tutte le modifiche da {destination}"
+
+#: lib/App/Sqitch/Engine.pm:308
+#, perl-brace-format
+msgid "Revert all changes from {destination}?"
+msgstr "Annullo tutte le modifiche da {destination}?"
+
+#: lib/App/Sqitch/Engine.pm:339
+#, perl-brace-format
+msgid "Verifying {destination}"
+msgstr "Veifico {destination}"
+
+#: lib/App/Sqitch/Engine.pm:347
+msgid "Nothing to verify (no planned or deployed changes)"
+msgstr ""
+"Niente da verificare (nessuna modifica pianificata o per cui effettuare "
+"deploy)"
+
+#: lib/App/Sqitch/Engine.pm:357
+msgid "There are deployed changes, but none planned!"
+msgstr "Ci sono modifiche incluse nel deploy, ma nessuna pianificata!"
+
+#: lib/App/Sqitch/Engine.pm:377
+msgid "Verify Summary Report"
+msgstr "Report Sommario di verifica"
+
+#: lib/App/Sqitch/Engine.pm:380
+#, perl-brace-format
+msgid "Changes: {number}"
+msgstr "Modifiche: {number}"
+
+#: lib/App/Sqitch/Engine.pm:381
+#, perl-brace-format
+msgid "Errors:  {number}"
+msgstr "Errori: {number}"
+
+#: lib/App/Sqitch/Engine.pm:382
+msgid "Verify failed"
+msgstr "Verifica fallita"
+
+#: lib/App/Sqitch/Engine.pm:387
+msgid "Verify successful"
+msgstr "Verifica riuscita"
+
+#: lib/App/Sqitch/Engine.pm:400
+#, perl-brace-format
+msgid "Change \"{change}\" has not been deployed"
+msgstr "Modifica \"{change}\" non deployata"
+
+#: lib/App/Sqitch/Engine.pm:403
+#, perl-brace-format
+msgid "Cannot find \"{change}\" in the database or the plan"
+msgstr "Non posso trovare \"{change}\" nel database o nel piano di esecuzione"
+
+#: lib/App/Sqitch/Engine.pm:410
+#, perl-brace-format
+msgid "Change \"{change}\" is deployed, but not planned"
+msgstr "La modifica \"{change}\" è deployata ma non pianificata"
+
+#: lib/App/Sqitch/Engine.pm:454
+msgid "Out of order"
+msgstr "Fuori servizio"
+
+#: lib/App/Sqitch/Engine.pm:460
+msgid "Not present in the plan"
+msgstr "Non presente nel piano di esecuzione"
+
+#: lib/App/Sqitch/Engine.pm:471 lib/App/Sqitch/Engine.pm:483
+#: lib/App/Sqitch/Engine.pm:963 lib/App/Sqitch/Engine.pm:993
+msgid "not ok"
+msgstr "Non OK"
+
+#: lib/App/Sqitch/Engine.pm:471 lib/App/Sqitch/Engine.pm:941
+#: lib/App/Sqitch/Engine.pm:983
+msgid "ok"
+msgstr "OK"
+
+#: lib/App/Sqitch/Engine.pm:485
+msgid "Not deployed"
+msgstr "Non deployato"
+
+#: lib/App/Sqitch/Engine.pm:517
+#, perl-brace-format
+msgid "Verify script \"{script}\" failed."
+msgstr "Script di verifica \"{script}\" fallito"
+
+#: lib/App/Sqitch/Engine.pm:526
+#, perl-brace-format
+msgid "Verify script {file} does not exist"
+msgstr "Lo script di verifica {file} non esiste"
+
+#: lib/App/Sqitch/Engine.pm:567
+#, perl-brace-format
+msgid "Conflicts with previously deployed change: {changes}"
+msgid_plural "Conflicts with previously deployed changes: {changes}"
+msgstr[0] ""
+"Ci sono conflitti con la modifica precdedentemente deployata: {changes}"
+msgstr[1] ""
+"Ci sono conflitti con le modifiche precdedentemente deployate: {changes}"
```

Comment:
"Ci sono conflitti con le modifiche **precedentemente** deployate: {changes}" [[also consider changing "deployate" to "implementate"]